Classes/Emogrifier.php

```diff
@@ -1501,4 +1505,34 @@ private function getNodesToExclude(\DOMXPath $xPath)
 
         return $excludedNodes;
     }
+
+    /**
+     * Handle warnings in process() method
+     *
+     * @param int $errNo
+     * @param string $errStr
+     * @param string $errFile
+     * @param int $errLine
+     * @param array $errContext
+     *
+     * @return bool
```

Comment:
Ah, okay, I didn't know that. Thanks for the explanation. Could you then please add something like "bool always false" plus an explanation to the return annotation? Thanks!